play-testkit/src/main/scala/play/api/test/SelfSigned.scala

```diff
@@ -24,16 +24,21 @@ object SelfSigned {
   lazy val (sslContext, trustManager): (SSLContext, X509TrustManager) = {
     val keyStore: KeyStore = FakeKeyStore.generateKeyStore
 
-    val kmf: KeyManagerFactory = KeyManagerFactory.getInstance("SunX509")
-    kmf.init(keyStore, "".toCharArray)
-    val kms: Array[KeyManager] = kmf.getKeyManagers
+    buildContextAndTrust(keyStore)
+  }
+
+  // TODO: this should be moved to `ssl-config`
+  private def buildContextAndTrust(keyStore: KeyStore) = {
 
-    val tmf: TrustManagerFactory = TrustManagerFactory
-      .getInstance(TrustManagerFactory.getDefaultAlgorithm())
+    val tmf: TrustManagerFactory = TrustManagerFactory.getInstance(TrustManagerFactory.getDefaultAlgorithm)
     tmf.init(keyStore)
     val tms: Array[TrustManager] = tmf.getTrustManagers
     val x509TrustManager: X509TrustManager = tms(0).asInstanceOf[X509TrustManager]
 
+    val kmf: KeyManagerFactory = KeyManagerFactory.getInstance(KeyManagerFactory.getDefaultAlgorithm)
```

Comment:
@ignasi35 was the change from "SunX509" to `KeyManagerFactory.getDefaultAlgorithm` intentional?

Reply:
Moreover, is it significant?

Reply:
It was intentional and _I think_ it's not significant.